README.md

```diff
@@ -117,8 +149,25 @@ spec:
       useExternalOpenvswitch: false
 ```
 
-## Configuring kube-proxy
-Users may customize the kube-proxy configuration. None of these settings are required:
+### Configuring OVNKubernetes
+OVNKubernetes supports the following configuration options, all of which are optional:
+* `genevePort`: The port to use for the geneve overlay. The default is 6081
+* `MTU`: The MTU to use for the geneve overlay. The default is the MTU of the node that the cluster-network-operator is first run on, minus 100 bytes for geneve overhead. If the nodes in your cluster don't all have the same MTU then you may need to set this explicitly.
+
+These configuration flags are only in the Operator configuration object.
+
+Example:
+```yaml
+spec:
+  defaultNetwork:
+    type: OVNKubernetes
+    ovnKubernetesConfig:
+      genevePort: 6081
+      mtu: 1400
+```
+
+## Configuring openshift-sdn kube-proxy
+Users may customize the openshift-sdn kube-proxy configuration. Ovn is a proxy and doesn't use kube-proxy. None of these settings are required:
```

Comment:
This isn't quite right. In theory the configuration here can be used for third-party plugins with a separate kube-proxy process as well, as discussed in the `deployKubeProxy` paragraph.

So we need to explain that
1. Some plugins (like OpenShift SDN) have a built-in kube-proxy, some plugins require a standalone kube-proxy to be deployed, and some (like OVN and Kuryr) don't use kube-proxy at all.
2. The `deployKubeProxy` flag can be used to indicate whether CNO should deploy a standalone kube-proxy, but for supported network types, this will default to the correct value automatically.
3. For plugins that use kube-proxy (whether built-in or standalone), you can configure the proxy via `kubeProxyConfig`

(If you split out a pre-4.0-fixes-and-clarifications-only PR, this change could be made there, since the idea of it is still correct even if we don't actually support OVN yet.)